OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs

```diff
@@ -604,7 +607,9 @@ void SelectServer(GameServer server)
 				foreach (var option in kv.Value)
 				{
 					var o = option;
-					var playerName = o.IsBot ? currentMap.GetMessage(o.Name) : o.Name;
+					var playerName = o.IsBot
```

Comment:
Should this not be dynamic too?

Reply:
it is dynamic, I only added cached transforms where it wasn't

Reply:
If you select a map, and then it updates afterwards, it won't be updating the names in the player list(?)

label.GetText below will keep the value that we determine here, before the map updates.

Reply:
it's an unlikely scenario, but handled it anyhow